File: coco/src/main/java/coco/ep/i1/finalized/Sub.java
package coco.ep.i1.finalized;

public class Sub extends Exp implements coco.ep.i1.Sub<Exp> {

    protected Exp left;
    protected Exp right;

    public Sub(Exp _left, Exp _right) {
        this.left = _left;
        this.right = _right;
    }

    public Exp getLeft() {
        return this.left;
    }
    public Exp getRight() {
        return this.right;
    }
}

File: coco_csharp/CoCo/EP/I1/Finalized/Sub.cs
namespace EP.I1.Finalized {
    public class Sub : Exp, EP.I1.Sub<Exp> {
        private EP.Exp<Exp> left;
        private EP.Exp<Exp> right;

        public Sub(EP.Exp<Exp> _left, EP.Exp<Exp> _right) {
            this.left = _left;
            this.right = _right;
        }

        public EP.Exp<Exp> getLeft() { return this.left; }

        public EP.Exp<Exp> getRight() { return this.right; }
    }
}